### DISCUSSION TITLE
add a newly added section

### DISCUSSION THREAD

**Discussion by OmiChrome** [yutkat/my-neovim-pluginlist#824] [Ideas]
make a section were  we can see what changes are made to list in a nice way , 

like newly added or somthign

**Comment by yutkat** [comment]
It is easy to update the RSS feed when there is an Add in the commit message, so I'll try to implement it next time I have time.

**Reply by yutkat** [reply]
feed: https://yutkat.github.io/new-neovim-plugin-with-category-rss/feed

https://github.com/yutkat/new-neovim-plugin-with-category-rss

**Reply by OmiChrome** [reply]
good
